### PR TITLE
Chapter1/homework/5 

### PR DESCRIPTION
**Pull request by draksa15**

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -9,7 +15,8 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0) return;
+            Array.Sort(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Implement your own sort.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -19,7 +26,8 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0) return;
+            Array.Reverse(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Implement your own reverse.

**Review by Almantask — CHANGES_REQUESTED** [review]
Please implement your own function and don't use C# language features yet.

**Comment by draksa15** [comment]
Fixed!

**Review by Almantask — APPROVED** [review]
Sorry for forgetting to check back! Looks good now!

**Pull request closed without merging**